.NET/Microsoft.Recognizers.Definitions.Common/Swedish/DateTimeDefinitions.cs

```diff
@@ -0,0 +1,753 @@
+﻿//------------------------------------------------------------------------------
```

Comment:
It would be best to keep only the Choice-related changes in this PR.

Reply:
Fixed